src/OrganisasjonsListeProvider.tsx

```diff
@@ -60,8 +60,7 @@ export const ListeMedAltinnSkjemaKoder: AltinnSkjema[] = [
     {
         navn: 'Arbeidstrening',
         kode: '5332',
-        versjon: '2',
-        testversjon: '1',
+        versjon: gittMiljo({prod: '2', other: '1'})
```

Comment:
@Oslove I stede for å sjekke testversjon rundt om riktig, så settes det riktig versjon her.

Reply:
Ok 👍